src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/SlimefunMachine.java

```diff
@@ -47,6 +62,10 @@ public SlimefunMachine(Category category, ItemStack item, String id, ItemStack[]
 		return this.shownRecipes;
 	}
 	
+	public static BlockFace[] getFaces() {
```

Comment:
Should probably be called getOutputFaces()

Reply:
in that case I'll change the field to outputFaces

Reply:
It's probably a better name, yeah.